editor/editor_file_system.cpp

```diff
@@ -1478,7 +1478,7 @@ void EditorFileSystem::_reimport_file(const String &p_file) {
 		Ref<ConfigFile> cf;
 		cf.instance();
 		Error err = cf->load(p_file + ".import");
-		if (err == OK) {
+		if (err == OK && cf->has_section("params")) {
```

Comment:
I haven't looked into details, but isn't the `importer_name = cf->get_value("remap", "importer");` call in the `if` still relevant even if the `params` section is missing?

Reply:
Yep, just caught that, making the necessary changes.